src/shared/creds-util.c

```diff
@@ -1046,6 +1048,9 @@ int decrypt_credential_and_warn(
                                 le32toh(t->blob_size),
                                 t->policy_hash_and_blob + le32toh(t->blob_size),
                                 le32toh(t->policy_hash_size),
+                                // TODO BILL WHAT TO PLUMB HERE
```

Comment:
i guess we need to extend the credential structure to make this work. can be done later. maybe add an explanatory comment here, and add an entry to the TODO file